aplib.net-demo/Assets/Scripts/Inventory/Item.cs

```diff
@@ -19,6 +19,11 @@ public abstract class Item : MonoBehaviour
     /// </summary>
     public int uses;
 
+    /// <summary>
+    /// The amount of uses the item starts with.
+    /// </summary>
+    public int startUses;
+
```

Comment:
This was renamedto "usesAddedPerPickup", since it gets added on every item pickup, not just the first pickup
```suggestion
```